frontend/src/libs/packages/store/middlewares/notification-middleware.ts

```diff
@@ -0,0 +1,31 @@
+import { type TypedStartListening } from '@reduxjs/toolkit';
+import { createListenerMiddleware, isRejected } from '@reduxjs/toolkit';
+
+import { type AsyncThunkConfig } from '~/libs/types/async-thunk-config.type';
+import { NotificationType } from '~/packages/notification/libs/enums/notification-type.enum';
```

Comment:
If it's cross-usable then it should be reexported on through root package level
```suggestion
import { NotificationType } from '~/packages/notification/notification.js';
```